### DISCUSSION TITLE
Swipe down to go back!

### DISCUSSION THREAD

**Discussion by 5haaad** [IacobIonut01/Gallery#407] [Ideas]
Hi,
I just found out about this work, it's a _great_ application so far but with some slight changes it would be the best gallery app. Beside that it's laggy sometime, it misses a **must-have** feature _(at least in my opinion)_ which is that _when you're on a picture and you want to go back with a single swipe down_ but it is not there, the only 2 ways to go back is to hit the back button on the top or to use the back gesture of the phone. So, I hope you add it, it is great feature to have...

**Comment by tomaszk8266** [comment]
It's now present in the 3.0.0 release
